plugins/destination/clickhouse/client/migrate.go

```diff
@@ -103,23 +142,21 @@ func unsafeChanges(changes []schema.TableColumnChange) []schema.TableColumnChang
 }
 
 func (c *Client) createTable(ctx context.Context, table *schema.Table, partition []spec.PartitionStrategy, orderBy []spec.OrderByStrategy) (err error) {
-	c.logger.Debug().Str("table", table.Name).Msg("Table doesn't exist, creating")
```

Comment:
This was confusing as we called `dropTable` then `createTable` when doing force migration, so even if the table existed this log was printed